### PR TITLE
Anton Edit views

### PR DESCRIPTION
**Pull request by alevel-petrenko**

### CODE REVIEW

File: Lib/Services/ServerService.cs [hunk 1]
```diff
@@ -10,15 +10,13 @@ namespace Lib.Services
...
     public class ServerService : IServerService
     {
-        private static string _path = @"C:\Users\evgen\Desktop\AUVebServers\db.txt";
+        private static string _path = @"E:\A_level\Git\AUVebServers\db.txt";
 
         public static List<Server> _db = new List<Server>() { };
 
         static ServerService()
         {
             //_db.Add(new Server { Id = 1, Domen = "www.test.com", Name = "Test name", Type = ServerType.Gaming });
```
Comment (merchansky-alevel) [inline Lib/Services/ServerService.cs:19]: In final megre we should remove commented code parts

File: UI/Controllers/ServersController.cs [hunk 2]
```diff
@@ -59,19 +59,26 @@ public async Task<ActionResult> Edit(int id)
             }
 
             return View(serverForUpdating);
+
+            //else?
```
Comment (merchansky-alevel) [inline UI/Controllers/ServersController.cs:63]: same

File: UI/Views/Servers/Edit.cshtml [hunk 3]
```diff
@@ -3,5 +3,12 @@
     ViewBag.Title = "Edit";
 }
 
-<h2>Edit</h2>
+<h2>Редактирование</h2>
```
Comment (merchansky-alevel) [inline UI/Views/Servers/Edit.cshtml:6]: 👍

File: UI/Views/Servers/Edit.cshtml [hunk 4]
```diff
@@ -3,5 +3,12 @@
...
 
+
+@using (Html.BeginForm("Edit", "Servers", FormMethod.Post))
+{
+    Html.RenderPartial("Partials/ServerFormPartials");
```
Comment (merchansky-alevel) [inline UI/Views/Servers/Edit.cshtml:11]: 👍

File: UI/Views/Servers/Edit.cshtml [hunk 5]
```diff
@@ -3,5 +3,12 @@
...
+
+    <button type="submit">Редактировать сервер</button>
```
Comment (merchansky-alevel) [inline UI/Views/Servers/Edit.cshtml:13]: 👍

File: UI/Views/Servers/Index.cshtml [hunk 6]
```diff
@@ -29,7 +29,7 @@
     <div class="row">
         <div class="col-md-12">
             <div style="text-align: right">
-                @Html.ActionLink("Добавть сервер", "Add", "Servers")
+                @Html.ActionLink("Добавить сервер", "Add", "Servers")
```
Comment (merchansky-alevel) [inline UI/Views/Servers/Index.cshtml:42]: 👍

File: UI/Views/Servers/Partials/ServerFormPartials.cshtml [hunk 7]
```diff
@@ -12,6 +12,6 @@
 </div>
 <p>Тип:</p>
 <div>
-    @Html.DropDownListFor(a => a.Type, new SelectList(Enum.GetValues(typeof(Lib.Models.ServerType))));
+    @Html.DropDownListFor(a => a.Type, new SelectList(Enum.GetValues(typeof(Lib.Models.ServerType))))
```
Comment (merchansky-alevel) [inline UI/Views/Servers/Partials/ServerFormPartials.cshtml:15]: 👍

File: UI/Views/Servers/Delete.cshtml [hunk 8]
```diff
@@ -0,0 +1,6 @@
+﻿
```
Comment (merchansky-alevel) [inline UI/Views/Servers/Delete.cshtml]: Unnecessary view

File: UI/Controllers/ServersController.cs [hunk 9]
```diff
@@ -59,19 +59,26 @@ public async Task<ActionResult> Edit(int id)
...
         }
         
         /// <summary>
         /// !REQUEST from updating form!
         /// </summary>
         /// <param name="server"></param>
-        [HttpPut]
-        public async Task<HttpStatusCode> Edit(Server server)
+        [HttpPost]
```
Comment (merchansky-alevel) [inline UI/Controllers/ServersController.cs:70]: For updating some model we should use PUT request, i think.

File: Lib/Services/ServerService.cs [hunk 10]
```diff
@@ -10,15 +10,13 @@ namespace Lib.Services
 {
     public class ServerService : IServerService
     {
-        private static string _path = @"C:\Users\evgen\Desktop\AUVebServers\db.txt";
+        private static string _path = @"E:\A_level\Git\AUVebServers\db.txt";
```
Comment (merchansky-alevel) [inline Lib/Services/ServerService.cs:13]: 👍

**Review by merchansky-alevel — APPROVED** [review]

File: UI/Controllers/ServersController.cs (hunk 9 above)
Comment (alevel-petrenko) [reply UI/Controllers/ServersController.cs:70]: attribute HttpPut doesn't work. and I received server's crash
meanwhile HttpPost works fine

Comment (merchansky-alevel) [reply UI/Controllers/ServersController.cs:70]: @alevel-petrenko , kk

**Pull request merged**